python/hail/dataset.py

```diff
@@ -5323,6 +5323,249 @@ def vep(self, config, block_size=1000, root='va.vep', csq=False):
 
     @handle_py4j
     @record_method
+    @typecheck_method(config=strlike,
+                      block_size=integral,
+                      root=strlike)
+    def nirvana(self, config, block_size = 500000, root = 'va.nirvana'):
```

Comment:
last change: PEP8 dictates no spaces around keyword args and default arguments, e.g. `root='va.nirvana'`.